en/flight_modes/altitude.md

```diff
@@ -0,0 +1,55 @@
+# ALTITUDE CONTROL Flight Mode
+
+The ALTITUDE CONTROL flight mode maintains constant altitude when all remote control inputs are centered (i.e. no roll, pitch, yaw and at 50% throttle). The mode does not use GPS, and hence will not attempt to hold position against wind.
+
+> **Tip** This mode is recommended for beginners because it is easy to return the vehicle to predictable flight.
+
+<span></span>
+> **Note**
+>  * This mode is *assisted*. The autopilot makes it easier for users to control the vehicle. 
+>  * This mode requires that a radio control is connected.
+>  * The altitude is normally measured using an altitude barometer, which may become inaccurate in extreme weather conditions. Vehicles that include a LIDAR/range sensor will be able to control altitude with greater reliability and accuracy. 
+<!-- ? * The vehicle must be armed before this mode can be engaged. -->
+
+The flight mode has additional characteristics based on the type of vehicle. These are discussed in the sections below. 
+
+
+## Multi-copter (MC)
+
+When all remote control inputs are centered (no roll, pitch, yaw, and ~50% throttle) the aircraft will *hover* at its current altitude (subject to wind).
+
+Throttle inputs control the *rate* of ascent/descent (i.e. the rate increases with distance from the stick centre), and there is a large deadzone at the centre. Roll, pitch and yaw inputs are fully controlled by the user (as in MANUAL mode).
+
+![Altitude Control MC](../../images/flight_modes/altitude_control_mode_copter.png)
+
+<!-- 
+what parameters affect MC behaviour - e.g. 

```

Comment:
1. what parameters affect MC behaviour - e.g. 
    - size of deadzone. 
    - maximum rate of ascend/descend 
2. - how is the behaviour tuned? (unless this is obvious from parameters.